packages/cloud-cognitive/src/components/UserProfileImage/UserProfileImage.js

```diff
@@ -0,0 +1,168 @@
+/**
+ * Copyright IBM Corp. 2021, 2021
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Canary } from '../_Canary';
+import { pkg } from '../../settings';
+const componentName = 'UserProfileImage';
+
+import {
+  User16,
+  User20,
+  User24,
+  User32,
+  Group16,
+  Group20,
+  Group24,
+  Group32,
+} from '@carbon/icons-react';
+
+//import { expPrefix } from '../../global/js/settings';
+import cx from 'classnames';
+
+const blockClass = `${pkg.prefix}-user-profile-avatar`;
+
+export const UserProfileImage = !pkg.isComponentEnabled(componentName)
```

Comment:
Sorry, the Canary mechanism has changed, after we found this was breaking Storybook. Have a look at ExampleComponent to see what's needed. The good news is it's a lot simpler :-) Basically:
 * no need to import Canary
 * use `export let` rather than `export const`
 * no other gorp needed -- straight into the component definition
 * then add the following line further down the file:
```js
// Return a placeholder if not released and not enabled by feature flag
UserProfileImage = pkg.checkComponentEnabled(UserProfileImage, componentName);
```

Reply:
Actually, let's merge this and I'll update the file -- I went through doing all the other components so I'm practised :-)